.github/workflows/main.yml

```diff
@@ -315,12 +319,16 @@ jobs:
       - name: run ${{ matrix.name }}
         run: make itest-parallel tranches=${{ env.TRANCHES }} ${{ matrix.args }} shuffleseed=${{ github.run_id }}${{ strategy.job-index }}
 
+      - name: Clean coverage
```

Comment:
Nice trick!!

Reply:
Yeah, I think with this trick we actually achieve higher combined coverage than before. But re-running everything to confirm.